manifests/Hyperhue-Studios.yaml

```
@@ -0,0 +1,13 @@
+name: HyperHue-Studios
+authors: ChemtrailOperator
+homepage: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky
+license: GPL-3.0-or-later
+version: v1.4
+shortDescription: In-Game access to all 29 vanilla swizzles.
+description: This plugin creates game play to swizzle your ship to any of the 29 vanilla swizzles. To begin, visit any human planet.
+url: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/releases/download/v1.4/HyperHue-Studios-v1.4.zip
+iconUrl: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/raw/main/v1.4/icon.png
+autoupdate:
+  type: tag
+  url: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/releases/download/$version/HyperHue-Studios-v1.4.zip
```

The current filename will work, but still needs this one change
```suggestion
  url: https://github.com/chemtrailoperator/HyperHue-Studios_EndlessSky/releases/download/$version/HyperHue-Studios-$version.zip
```